package-lock.json

```diff
@@ -1,10323 +1,8 @@
 {
   "name": "@cpanel/api",
   "version": "5.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Any reason for the lockfileversion change?

Reply:
According to https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#lockfileversion v2 is generated by npm 7+ and v1 is from npm 5 and 6. The first LTS version of node to get npm 7 by default was node 16, so that may be the cause.

Reply:
Good to know, thanks Aneece!